ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -39,14 +39,7 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
-- as: installer-rehearse-aws-disconnected-private-usgov
```

Comment:
@yunjiang29 do you want this job get removed in this PR?

Reply:
yes, I'd like to keep one rehearse job, if more jobs are required for testing,  I'll add them.